### ISSUE TITLE
Proposal: ability to add description/notes to each app

### ISSUE THREAD

**Issue by zinuzoid** [deis/workflow#597]
I have 100+ app running in my cluster. It's quite hard to maintenance each app(in perspective of administrator) if you don't know where is this app from, what does it do, which git repo it use, ... Currently, I have to create separate spreadsheet outside to track this information.

I propose to have new cmd to add a description as a general strings to each app for administration purpose.

### See new proposal in next comment
```
$deis apps:description "This is a coin mining worker. Check https://github.com/deis/mining"

$deis apps:info
=== test Application
updated:  2016-11-04T13:17:00Z
uuid:     ef2379cd-ba3a-4034-96c4-a23057111a9d
created:  2016-11-01T08:26:36Z
url:      test.deis.com
owner:    jim
id:       test
description: This is a coin mining worker. Check https://github.com/deis/mining

=== test Processes
--- cmd:
test-cmd-3183491288-lchzc up (v4)

=== agg-gta-live Domains
test
```

Possibly add this information to new field in `api_app` table

I can contribute this PR also.

**Comment by rimusz** [comment]
sounds as a good feature 👍

**Comment by intellix** [comment]
Maybe related: https://github.com/deis/workflow/issues/545

**Comment by bacongobbler** [comment]
Perhaps a `deis labels` would be a suitable alternative for both use cases?

**Comment by bacongobbler** [comment]
Whoops, fat-fingered the close button sorry

**Comment by zinuzoid** [comment]
I like `deis labels` idea. thanks @bacongobbler 

Here's new propose:
```
$deis labels:set project=alice git_repo=git://deis/mining.git

$deis labels:list
=== test Labels
project: alice
git_repo: git://deis/mining.git

$deis labels:unset project git_repo
```

on `labels:set` will update 2 things
- deis database
- kubernetes deployment label tag

I'll wait for some LGTM before I go ahead then.

**Comment by kmala** [comment]
This looks good and i feel this is something which can go into the appsettings which doesn't create a new release or do you think any specific reason to create a release.
 @bacongobbler @mboersma can shed more light on the proposal.

**Comment by bacongobbler** [comment]
If we're not touching the manifests at all and just adding labels to each app for querying purposes I don't see a reason why we should roll a new release. I agree @kmala. :)

**Comment by zinuzoid** [comment]
Ok. so I'll go ahead with cli format as the last comment, but will update only in the database. And it'll be a new field `label` in api_appsetting table with json object value.

**Comment by kmala** [comment]
Closing the issue since all the PR's are merged and part of the 2.9.0 release.
